src/components/SoftwareVersion.tsx

```diff
@@ -0,0 +1,30 @@
+import React, {useEffect, useState} from "react";
+import SoftwareVersionFetcher from "../minecraft-versioning/SoftwareVersionFetcher";
+
+export default function SoftwareVersion({ versionType, project = "paper" }) {
+    const [fetched, setFetched] = useState(null);
```

Comment:
```suggestion
    const [fetched, setFetched] = useState<string>();
```

My LSP complains about the typing being wrong here, and this seems more correct.